src/System.Net.Security/src/System/Net/Security/SslApplicationProtocol.cs

```diff
@@ -34,7 +34,7 @@ internal SslApplicationProtocol(byte[] protocol, bool copy)
             {
                 byte[] temp = new byte[protocol.Length];
                 Array.Copy(protocol, temp, protocol.Length);
```

Comment:
I realize this is pre-existing, but as long as you're changing this code, this line would be better as:
```C#
Array.Copy(protocol, 0, temp, 0, protocol.Length);
```

Reply:
Done